.project

```diff
@@ -0,0 +1,34 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
[medium] I would rather not commit this file as well since it should be taken care of by maven.